projects/js-toolkit/packages/portal-base/package.json

```diff
@@ -6,6 +6,7 @@
 		"@babel/preset-react": "^7.13.13",
 		"@liferay/js-toolkit-core": "^5.0.0",
 		"babel-loader": "^8.2.3",
+		"babel-minify": "^0.5.2",
```

Comment:
Should this be `babel-preset-minify`?

Reply:
Yes. How did it work, then? :open_mouth:

Reply:
Maybe some transitive dependency?

Reply:
Apparently babel-minify depends on babel-preset-minify, and not the opposite. Weird...

![image](https://user-images.githubusercontent.com/3273630/183876948-edbb5bc5-5c7f-455f-a074-2daad6dbe975.png)

Reply:
OK, that's because `babel-minify` is a CLI, not the babel plugin (which is `babel-preset-minify`)

Reply:
Fixed!